server/roomlogs.ts

```diff
@@ -16,6 +17,23 @@ interface RoomlogOptions {
 	noLogTimes?: boolean;
 }
 
+
+interface RoomlogRow {
+	type: string;
+	room: string;
+	user: string | null;
+	day: string;
+	month: string;
+	year: string;
+	timestamp: number;
+	message: string;
+	// tsvector, really don't use
+	content: string | null;
+}
+
+export const roomlogDB = Config.replaysdb ? new PGDatabase(Config.replaysdb) : null!;
```

Comment:
Why `null!`?

Reply:
(removed, but for posterity I had copied it from Rooms.Replays)